doc/src/build/wallet.md

```diff
@@ -784,10 +784,13 @@ public(script) fun transfer(c: Coin::Coin<SUI>, recipient: address, _ctx: &mut T
 ```
 
 Please note that there is no real need to use a Move call to transfer
-objects as this can be accomplish with a built-in wallet
-[command](#transferring-objects) - we chose this example due to its
+objects as this can be accomplished with a built-in wallet
```

Comment:
```suggestion
coins as this can be accomplished with a built-in wallet
```

Reply:
Also I think the paragraph in line 511 is strange: if there are no objects, there are also no gas, which means no transactions can be done. This part of doc needs to be updated? @patrickkuo